packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -40,62 +47,73 @@ interface UserHeaderItemProps {
 
 export function UserHeaderItem(props: PropsWithChildren<UserHeaderItemProps>) {
     const { aboutModalLogo, buttonChildren, children } = props;
-
     const theme = useTheme();
     const isMobile = useMediaQuery(theme.breakpoints.down("md"));
-
     const user = useCurrentUser();
+
     const [showAboutModal, setShowAboutModal] = useState(false);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
-    const AccountIcon = user.impersonated ? <Account color="info" /> : <Account />;
-
     return (
-        <AppHeaderDropdown buttonChildren={buttonChildren ?? (isMobile ? AccountIcon : user.name)} startIcon={isMobile ? undefined : AccountIcon}>
-            <DropdownContent padding={4}>
-                <Button
-                    fullWidth={true}
-                    startIcon={<Info />}
-                    onClick={() => {
-                        setShowAboutModal(true);
-                    }}
-                    color="info"
-                >
-                    <FormattedMessage id="comet.about" defaultMessage="About" />
-                </Button>
-                {children}
+        <AppHeaderDropdown
+            buttonChildren={buttonChildren ?? (isMobile ? <UserHeaderAccountIcon impersonated={user.impersonated} /> : user.name)}
+            startIcon={isMobile ? undefined : <UserHeaderAccountIcon impersonated={user.impersonated} />}
+        >
+            <DropdownContent padding={0}>
+                <Box padding={4}>
```

Comment:
Please use either styled components or Box directly, but not both. @jamesricky what do you prefer?

Reply:
IMO `Box` is fine if it's just one or two styling properties, if there is more styling I'd prefer a styled component.